models/tokenprice/tokenprice_swap_vwap_daily.sql

```diff
@@ -0,0 +1,83 @@
+{{
+    config(
+        materialized='incremental',
+        unique_key="block_date||'-'||token_address",
+        incremental_strategy = 'delete+insert',
+        tags=['core', 'swaps', 'defi','tokenprice'],
+        cluster_by=['block_date', 'token_address']
+        )
+}}
+
+with
+
+-- Assuming these stables have USD value of 1
+stables as (
+    select 
+        token_address,
+        token_symbol,
+        decimals
+    from tokens 
+    where token_symbol in  ('1USDC', '1USDT', '1DAI', 'UST', 'BUSD')
+),
+
+swaps1 as (
+    select 
+        log_id,
+        block_timestamp,
+        date_trunc('day',block_timestamp) as block_date,
+        tx_hash,
+        token0_address as token_address,
+        token0_symbol as token_symbol,
+        (amount0In + amount0Out) as token_amount_raw,
+        tokens.decimals as token_decimals,
+        ((amount0In + amount0Out) / pow(10, tokens.decimals)) as token_amount_norm,
+        (amount1In + amount1Out) as stable_amount_raw,
+        stables.decimals as stable_decimals,
+        ((amount1In + amount1Out) / pow(10, stables.decimals)) as stable_amount_norm
+    from swaps
```

Comment:
the tokenprice_swap_vwap_daily model was my attempt at pulling all the prices from direct dex swaps, however most markets were too illiquid to be usable.  Instead of using this approach, i used tranquil_markets_tokenprice which is derived from the most liquid pool path i could find to a stable (e.g. for stONE, it's stONE -> ONE -> USDC, but for ETH it's 1ETH -> JEWEL -> USDC)

We could (should) remove this model for this commit and no functionality will be impacted.